lib/fathead/sqlalchemy/parse.py

```diff
@@ -1,117 +1,204 @@
+
+            # Loop through the 'dd' tag in the current 'dl' tag 
+            for elem in curr_dd:
+                if elem is None or str(type(elem))=="<class 'bs4.element.NavigableString'>":# Skip the 'None' elements and non-tags
+                    continue
+                if elem.name=='dl':# Break if you come across a new 'dl' tag inside the current
+                    break
+                else:
+                    if elem.name=='div' and ('highlight-default' in elem['class'] or ('event-signatures' in elem['class'])) :
+                        strg=''
+			if 'event-signatures' in elem['class']:# Add event signatures separately
+			    ptag = elem.find('p')
+			    p+=ptag.text
+			    divtag= elem.find('div',{'class':'highlight-default'})
+			    strg+=divtag.text
 			else:
-				description=description.format(txt)
-			description = description.replace('\n','\\n')
-			code = code.replace('\n','\\n')
-			code = code.replace('\\\\n','\\n')
-			prog_container = prog_container.format(description+code)
-			
-
-			if title is not None:
-				title = title.encode('ascii','ignore')
-				title = title.lower()
-				ls = list(title)
-				ls[0] = ls[0].upper()
-				title = ''.join(ls)
-				file.write(title)
-
-			file.write('	')
-			file.write('A')
-			file.write('	')
-			file.write('	')
-			file.write('	')
-			file.write('	')
-			file.write('	')
-			file.write('	')
-			file.write('	')
-			file.write('	')
-			file.write('	')
-			file.write('	')
-			file.write(prog_container)
-			file.write('	')
-			file.write('http://docs.sqlalchemy.org/en/latest/orm/tutorial.html')
-			file.write('\n')
-
-		file.close()
-
+			    strg += elem.text
+			p = self.replace_unicodes(p)# Replace all unicodes with HTML equivalent
+                        strg = self.replace_unicodes(strg)# Replace all unicodes with HTML equivalent
+			txt += desc.format(p)+code.format(strg)
+			p=''
+                        code_done=1
+                    elif elem.name=='div' and 'seealso' in elem['class']:
+                        break# Ignore the divs which have 'see also' content
+                    else:
+                        if elem.name is not None:
+                            p+=elem.text
+			    code_done=0
+                        else:
+                            p+=elem
+			    code_done=0
+
+
+            cur_dt = k.find('dt')
+            
+            # Initialize and find the value of the current main topic
+            head = ''
+            for d in cur_dt:
+                if d is None or str(type(d))=="<class 'bs4.element.NavigableString'>":
+                    head+=d.string
+                    continue
+                if d.name=='a' and d['class'][0]=='headerlink':
+                    break
+                head+=d.string
+            
+            # Replace unicodes with HTML equivalent in the head
+            head= head.replace(u'\u2192','&#8594;')
+            head = head.replace('\n','\\n')
+            head= head.replace('\t','\\t')
```

Comment:
It looks like there aren't any in the output.txt so you can actually remove this line 👍